recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ test:
     - sksparse
     - sksparse.cholmod
   commands:
-    - nosetests -sv sksparse
+    - OPENBLAS_NUM_THREADS=1 nosetests -sv sksparse
```

Comment:
This is only an issue on CircleCI. It runs totally fine on my machine - its just hanging on CircleCI